ui/round/src/plugins/sanWriter.ts

```diff
@@ -1,30 +1,32 @@
-function fixCrazySan(san) {
+type Board = {pieces: {[key: number]: string}, turn: boolean, k?: number, K?: number};
```

Comment:
the `k?: number, K?: number` feels wrong

but on line 43 we have this `if (v === 'k' || v === 'K') board[v] = square;`

maybe that line is a mistake?

Reply:
can you find where in the code `board.k` or `board.K` is used?

Reply:
Looks like nowhere, tracing `board` through these various files.

Anywho, can leave it to someone else since left `noImplicitAny` off here.

Reply:
Probably in the original SAN writer for ceval output (to have to king positions for check), but not here :+1: